### ISSUE TITLE
Play/Pause state problem while going in full screen in lyrics mode

### ISSUE THREAD

**Issue by Gauravchaubey66** [fast4x/RiMusic#1417]
### Steps to reproduce the bug

1. Play any music which have lyrics
2. Show synced lyrics
3. Expand lyrics or do in full screen without stopping music 
4. There are play or pause button on the bottom of lyrics 
5. Even song is playing initially that button is at play mode (showing that music is not playing) 
6. When I click on that that goes into normal state again 
7. 

https://github.com/fast4x/RiMusic/assets/149640114/3e9de398-d329-4c5a-9c21-56b5674b9dce

### Expected behavior

It's should be on pause state.(it's mean music is playing that time)

### Actual behavior

It should be on play state(It's mean should shows that music is playing)

### RiMusic version

0.6.28.1

### Android version

Android 13

**Comment by aneesh1122** [comment]
Not happening in my case. Uninstall the version you have and download the latest version.

**Comment by fast4x** [comment]
This bug was solved in 0.6.29, update please..

**Comment by Gauravchaubey66** [comment]
Yeah, I installed new version as someone suggested not that thing is fixed now. but keep in mind that not every user are getting this update they have to go manually to install this update(from GitHub) because this update is not reflecting on fdroid, droidfy or neo store I checked it already.

**Comment by aneesh1122** [comment]
> Yeah, I installed new version as someone suggested that thing is fixed now (thanks to team). but keep in mind that not every user are getting this update they have to go manually to install this update(from GitHub) because this update is not reflecting on fdroid, droidfy or neo store I checked it already.

Well, you'll just have to come to GitHub and install it after every update. If you could come to GitHub and post a 'Issue' I'm sure you can download the app from here.

Or maybe update the app from GitHub before posting any issues? The app already notifies you when a new update is available. The app is free and the dev don't owe anyone anything tbh.

tldr : update the app from GitHub before posting any issues

**Comment by Gauravchaubey66** [comment]
I know that I can come but all user can't I did not post that issue for my individual problem. I did so that Rimusic team check this thing and fix.

And as you told I can update but I thing you should also know that only GitHub user are not user of Rimusic app.

**Comment by aneesh1122** [comment]
> I know that I can come but all user can't I did not post that issue for my individual problem. I did so that Rimusic team check this thing and fix.
> 
> And as you told I can update but I thing you should also know that only GitHub user are not user of Rimusic app.
> 
> And you mentioned that app already notified that's true but a normal user can update by going GitHub. So they just ignore this things and adjust with app I know that is not a big thing but overall it was reflecting on my device and I just post the issue keeping in mind that it can help developers of Rimusic.

1) Developer can't help you if you're not using the latest version and what's there to check and fix? You've posted the problem which is already solved. Not Dev's problem if you're not using the latest version.

2) The app is free and the dev is doing enough already. If the people ain't using GitHub and updating to the latest version then who is at fault here?

3) how it can help developers of Rimusic? He did his part already. He was aware of the bug and fixed it. how is it his fault that people are not using latest version?

Why did you post the issue in the first place? You are aware of GitHub and can easily download the latest version yet you decided to use the outdated version and posted an Issue which is solved already. Did you even try the latest version before posting this issue? Please install the latest update first before posting anything here.

There are enough pending genuine issues here already.

**Comment by Gauravchaubey66** [comment]
> > I know that I can come but all user can't I did not post that issue for my individual problem. I did so that Rimusic team check this thing and fix.
> > 
> > And as you told I can update but I thing you should also know that only GitHub user are not user of Rimusic app.
> > 
> > And you mentioned that app already notified that's true but a normal user can update by going GitHub. So they just ignore this things and adjust with app I know that is not a big thing but overall it was reflecting on my device and I just post the issue keeping in mind that it can help developers of Rimusic.
> 
> 1) Developer can't help you if you're not using the latest version and what's there to check and fix? You've posted the problem which is already solved. Not Dev's problem if you're not using the latest version.
> 
> 2) The app is free and the dev is doing enough already. If the people ain't using GitHub and updating to the latest version then who is at fault here?
> 
> 3) how it can help developers of Rimusic? He did his part already. He was aware of the bug and fixed it. how is it his fault that people are not using latest version?
> 
> Why did you post the issue in the first place? You are aware of GitHub and can easily download the latest version yet you decided to use the outdated version and posted an Issue which is solved already. Did you even try the latest version before posting this issue? Please install the latest update first before posting anything here.
> 
> There are enough pending genuine issues here already.
> 

I think you should read my previous comment carefully. Thanks for you suggestion that get update (from GitHub) before posting any issue. ❤

**Comment by fast4x** [comment]
Don’t worry, RiMusic has 3 publishing channels, the most updated is github while IzzyOnDroid updates itself either the same day or the next day. F-Droid users instead have to wait for the publication cycle that goes from one to two weeks after the official release on github.

This is to say that the version in F-Droid can keep bugs that have been fixed for a week or two. The solution is to keep the bug or update the app from github or IzzyOnDroid.

Thanks to all
